### PR TITLE
dontcheckhostname verify false

### PR DESCRIPTION
**Pull request by JudahSchwartz**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by JudahSchwartz** [comment]
https://code.pan.run/xsoar/content/-/pipelines/5224036

**Comment by JudahSchwartz** [comment]
https://code.pan.run/xsoar/content/-/pipelines/5224036

**Comment by JudahSchwartz** [comment]
@ilaner looks like an issue w precommit w py2

**Review by tkatzir — APPROVED** [review]
Nice and clean fix! :-)

File: Packs/Base/ReleaseNotes/1_32_0.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+##### CommonServerPython
+
+- Setting the a SSL Verify false argument will now set `check_hostname` to false as well.
```
Comment (JudahSchwartz) [inline Packs/Base/ReleaseNotes/1_32_0.md:3]:
```suggestion
- Setting the SSL Verify argument to false will now set `check_hostname` to false as well.
```

**Comment by JudahSchwartz** [comment]
forcing as xsoarng is failing due to a bw compatibility break on the server side. opening a bug

**Pull request merged**